### PR TITLE
Docs references

### PR DESCRIPTION
**Pull request by eysi09**

### CODE REVIEW

File: garden-service/src/commands/update-remote/modules.ts [hunk 1]
```diff
@@ -35,8 +35,8 @@ export class UpdateRemoteModulesCommand extends Command<Args> {
   arguments = updateRemoteModulesArguments
 
   description = dedent`
-    Remote modules are modules that have a repositoryUrl field
-    in their garden.yml config that points to a remote repository.
+    This command updates remote modules, i.e. modules that have a \`repositoryUrl\` field
```
Comment (drubin) [inline garden-service/src/commands/update-remote/modules.ts]: `This command` feels a little awkward when displaying help text I think the general context provides which command it's about?

Maybe? 
```
Updates remote modules, i.e. modules that have a \`repositoryUrl\` field...
```

Comment (thsig) [reply garden-service/src/commands/update-remote/modules.ts]: Yeah, agree with that. E.g. the description shown in `git help status` (on my machine) starts with

> Displays paths that have differences between...

and not

> This command displays paths that have differences between...

File: garden-service/src/docs/commands.ts [hunk 2]
```diff
@@ -19,7 +19,7 @@ import { describeParameters } from "../commands/base"
 
 export function generateCommandReferenceDocs(docsRoot: string) {
   const referenceDir = resolve(docsRoot, "reference")
-  const outputPath = resolve(referenceDir, "commands.md")
+  const outputPath = resolve(referenceDir, "command-reference.md")
```
[the remark this answers is not in the document]
Comment (eysi09) [reply garden-service/src/docs/commands.ts]: cc @ellenkorbes 

I changed the code to match the current file names. Alternatively we can revert the filename changes and leave the code as is.

File: garden-service/src/commands/update-remote/modules.ts (hunk 1 above)
Comment (eysi09) [reply garden-service/src/commands/update-remote/modules.ts]: Yes, agreed. All instances of `This command...` have been removed.

File: garden-service/src/docs/commands.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (drubin) [reply garden-service/src/docs/commands.ts]: I think it makes sense to revert the file name changes it makes ugly urls. But let's do that in another PR?

**Comment by drubin** [comment]
@eysi09 I think the changes are an improvement, but we need to fix the CI linter before merging.

**Pull request merged**